Samples/ProviderTelemetryComplete.tests.json

```
@@ -88,7 +88,7 @@
     {
         "FeatureFlagName": "Complete",
         "Inputs": {
-            "user": "Rachel",
+            "Users": "Rachel",
             "group": "beta"
```

```suggestion
            "Groups": ["beta"]
```
This seems to be a bug from my PR.

---

Oh thanks- I'll resolve it